test/integration/k8s_test.go

```diff
@@ -188,7 +188,7 @@ func TestPodScaling(t *testing.T) {
 			defer cancel()
 
 			pfOpts := PortForwardingOpts{
-				Namespace:     "default",
+				Namespace:     "linux-datapath-test",
```

Comment:
The test is for pod scaling. and namespace is states otherwise . Can we parameterize this if possible.